static/tests/frontend/specs/enable_disable.js

```diff
@@ -3,9 +3,12 @@
 
 describe('enable and disable webrtc', function () {
   context('WebRTC is disabled', function () {
-    before(async function () {
-      await helper.newPad({
+    before(function (done) {
```

Comment:
@JohnMcLear One of your lining passes lead to this regression. Previously it just had `cb: done`. Maybe the linter thought that it wasn't doing anything. So here, I made it do something, which is wait for padChrome to load. May not be strictly necessary but logically it's what we're doing anyway so we might as well. And I'm hoping it'll be acceptable to the linter now. But that's based on a guess about what the problem was.